qa/logging-spi/build.gradle

```diff
@@ -0,0 +1,18 @@
+
+apply plugin: 'elasticsearch.base-internal-es-plugin'
+apply plugin: 'elasticsearch.internal-java-rest-test'
+
+esplugin {
+  name 'logging-spi-test'
+  description 'An test plugin to test the SPI behaviour of ES logging'
+  classname 'org.elasticsearch.test.logging.plugin.TestLoggingPlugin'
+}
+
+dependencies {
+  clusterPlugins project(':qa:logging-spi')
+}
+
+tasks.named("javadoc").configure {
+  // There seems to be some problem generating javadoc on a QA project that has a module definition
+  enabled = false
```

Comment:
@mark-vieira I had disable javadoc on this QA project because it couldn't the `org.elasticsearch.server` module while trying to parse `module-info.java`.
It looks like this is the first QA project with a `module-info`, so took the easy option to resolve it.